package.json

```diff
@@ -38,7 +38,7 @@
     "strapi"
   ],
   "dependencies": {
-    "axios": "^0.18.0",
+    "axios": "0.21.1",
```

Comment:
Why is the axios version updated?

Reply:
There were vulnerabilities found in some of axios 0.18.0 dependencies... So I think it's good if we could proceed with using the newer version of aixos!!